File: server_api/src/main/java/indi/zhuyst/security/pojo/AccessToken.java
package indi.zhuyst.security.pojo;

import io.swagger.annotations.ApiModelProperty;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * 权限Token，包含过期时间以及当前用户对象
 * @author zhuyst
 */
@Data
@NoArgsConstructor
public class AccessToken {

    /**
     * AccessToken
     */
    @ApiModelProperty("AccessToken")
    private String token;

    /**
     * 过期时间
     */
    @ApiModelProperty("过期时间")
    private Long expire;

    /**
     * Token对应用户对象
     */
    @ApiModelProperty("Token对应用户对象")
    private SecurityUser user;
}

File: client_windows/SkyBlog/SkyBlog.Model/Business/AccessToken.cs
namespace SkyBlog.Model.Business
{
    /// <summary>
    /// 授权令牌
    /// </summary>
    public class AccessToken
    {
        /// <summary>
        /// 令牌字符串，应放入Header中
        /// </summary>
        public string Token { get; set; }

        /// <summary>
        /// 过期时间
        /// </summary>
        public int Expire { get; set; }

        /// <summary>
        /// 授权的用户对象
        /// </summary>
        public User User { get; set; }
    }
}
